WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -67,7 +68,8 @@ public partial class Arena : PeriodicRunner
 	public event EventHandler<InputAddedEventArgs>? InputAdded;
 
 	public HashSet<Round> Rounds { get; } = new();
-	private IEnumerable<RoundState> RoundStates { get; set; } = Enumerable.Empty<RoundState>();
+	internal ImmutableList<RoundState> RoundStates { get; set; } = ImmutableList<RoundState>.Empty;
```

Comment:
I understand this but in this repo this is confusing. We usually do this only for testing, otherwise, it is public.

Reply:
Same thoughts, if no other way to solve this and we hardly go against making it `public` its ok.

Reply:
Done